src/operator/bufferToggle.ts

```diff
@@ -44,18 +46,18 @@ import {errorObject} from '../util/errorObject';
  * @owner Observable
  */
 export function bufferToggle<T, O>(openings: Observable<O>,
-                                   closingSelector: (value: O) => Observable<any>): Observable<T[]> {
+                                   closingSelector: (value: O) => SubscribableOrPromise<any> | void): Observable<T[]> {
```

Comment:
It's going to allow/encourage TypeScript users to use this method incorrectly. `void` is not an okay return here. They have to return an `Observable` if they don't throw. If we must, we must, but that's _really_ gross.